workout-tracker/src/views/UserPage/UserPage.css

```diff
@@ -1,26 +1,19 @@
-.dashboard-container {
-  display: flex;
-  position: fixed;
-  bottom: 0;
-  height: 94%;
+.content.ant-layout-content {
+  margin: 0 !important;
+  padding: 0 !important;
```

Comment:
You might want to remove the `!important` as it is considered bad practice as highlighted in [this article on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
I'm aware, however it was previously not overriding default ant styles. It seems to be unnecessary now so removed.